package.json

```diff
@@ -102,17 +102,16 @@
     "react-map-gl": "^3.3.0",
     "react-markdown": "^4.0.6",
     "react-modal": "^3.1.10",
-    "react-onclickoutside": "https://registry.npmjs.org/react-onclickoutside/-/react-onclickoutside-6.7.1.tgz",
+    "react-onclickoutside": "^6.7.1",
     "react-palm": "^1.1.2",
     "react-redux": "^4.0.0",
-    "react-tooltip": "https://registry.npmjs.org/react-tooltip/-/react-tooltip-3.3.0.tgz",
+    "react-tooltip": "^3.9.2",
     "react-vis": "^1.8.0",
     "redux": "^3.0.4",
     "redux-actions": "^2.2.1",
     "reselect": "^3.0.1",
-    "styled-components": "^2.2.4",
     "supercluster": "^3.0.0",
-    "type-analyzer": "https://registry.npmjs.org/type-analyzer/-/type-analyzer-0.2.1.tgz",
```

Comment:
the reason url is specified here is because there is also `unpm` version of these libraries. And unpm install will find those. Not sure if it is still the case. But I would preview leave the `url` here, so they don't break things

Reply:
I believe it's no longer the case. Just checked via doing `npm info type-analyzer`